packages/core/src/graphql/middleware.ts

```diff
@@ -35,37 +46,42 @@ export const graphql = (
     maxOperationAliases: 30,
   },
 ) => {
-  let yoga: YogaServerInstance<any, any> | undefined = undefined;
+  const graphqlSchema = buildGraphQLSchema(schema);
```

Comment:
So, if this errors (which it can) that will bubble up when we execute the API functions during the build, correct?

Want to make sure we aren't running these effects only when the user hits the endpoint.

Reply:
Correct